File: app/src/main/java/zhan/whitetoolbar/MainActivity.java
package zhan.whitetoolbar;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

  @Override protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
  }

  public void whiteClick(View view) {
    launchActivity(WhiteToolbarActivity.class);
  }

  public void whiteImageClick(View view) {
    launchActivity(WhiteToolbarToImageActivity.class);
  }

  public void whiteMiClick(View view) {
    launchActivity(WhiteMiToolbarActivity.class);
  }

  private void launchActivity(Class clazz) {
    Intent intent = new Intent(this, clazz);
    startActivity(intent);
  }
}

File: app/src/main/java/zhan/whitetoolbar/MainActivity.kt
package zhan.whitetoolbar

import android.content.Intent
import android.os.Bundle
import android.view.View

import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun whiteClick(view: View) {
        launchActivity(WhiteToolbarActivity::class.java)
    }

    fun whiteImageClick(view: View) {
        launchActivity(WhiteToolbarToImageActivity::class.java)
    }

    fun whiteMiClick(view: View) {
        launchActivity(WhiteMiToolbarActivity::class.java)
    }

    private fun launchActivity(clazz: Class<*>) {
        val intent = Intent(this, clazz)
        startActivity(intent)
    }
}
